### ISSUE TITLE
unable to retrieve child resources EAI_AGAIN 

### ISSUE THREAD

**Issue by daviburg** [microsoft/AzureStorageExplorer#5778]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.24.1

### Architecture

i86

### Storage Explorer Build Number

20220510.2

### Platform

Windows

### OS Version

Windows 11

### Bug Description

Unable to list the blob containers in storage explorer while connecting with account access key 1.

### Steps to Reproduce

1. Launch storage explorer
2. Attached container by name and using access key2 from azure management portal
3. Expanded attached storage blob containers to get list of containers

### Actual Experience

Unable to retrieve child resources.

Details:
{
  "name": "FetchError",
  "message": "request to https://saphostingerlabwestus2di.blob.core.cloudapi.de/?comp=list&include=metadata failed, reason: getaddrinfo EAI_AGAIN saphostingerlabwestus2di.blob.core.cloudapi.de",
  "stack": "FetchError: request to https://saphostingerlabwestus2di.blob.core.cloudapi.de/?comp=list&include=metadata failed, reason: getaddrinfo EAI_AGAIN saphostingerlabwestus2di.blob.core.cloudapi.de\n    at ClientRequest.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\node-fetch\\lib\\index.js:1491:11)\n    at ClientRequest.emit (node:events:390:28)\n    at TLSSocket.socketErrorListener (node:_http_client:447:9)\n    at TLSSocket.emit (node:events:390:28)\n    at emitErrorNT (node:internal/streams/destroy:157:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:122:3)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)",
  "type": "system",
  "errno": "EAI_AGAIN",
  "code": "EAI_AGAIN"
}

### Expected Experience

List the containers

### Additional Context

While the expand error is common, I did not see information related to getaddrinfo EAI_AGAIN. Troubleshooting guide about AD issue does not appear to apply.

**Comment by MRayermannMSFT** [comment]
@daviburg  can you give using system proxy setting a try? You can read more about it here: https://docs.microsoft.com/en-us/azure/storage/common/storage-explorer-network#use-system-proxy-preview

**Comment by MRayermannMSFT** [comment]
Also, @daviburg, what Azure cloud is your data in. Is it in the old Azure Germany/Blackforest cloud? Asking because I see the `.de` on your endpoints. As an FYI, that Azure cloud has been shutdown....

**Comment by daviburg** [comment]
Updated storage explorer to the new current latest. Getting a better error message now:

Unable to retrieve child resources.

```
Details:
{
  "name": "Error",
  "message": "net::ERR_NAME_NOT_RESOLVED",
  "stack": "Error: net::ERR_NAME_NOT_RESOLVED\n    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:105:7047)\n    at SimpleURLLoaderWrapper.emit (node:events:390:28)"
}
```
Which is consistent with the unexpected endpoint suffix from a defunct cloud.

I removed the old connection and re-added it. This time it picks the right cloud. I wonder if the portal was temporarily pointing to Blackforest a month ago? Maybe internal outage tracking system for storage service will have some record of such incident.

(Ok to close).
